src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projectpage/view/ProjectPage.kt

```diff
@@ -120,35 +120,24 @@ class ProjectPage : View() {
                 }
             }
         }
-
         // Plugin active cover
-        stackpane {
-            style {
-                alignment = Pos.CENTER
-                backgroundColor += Color.BLACK
-                        .deriveColor(0.0, 0.0, 0.0, 0.5)
-            }
-            val icon = MaterialIconView(MaterialIcon.MIC_NONE, "60px")
-                    .apply {
-                        style(true) {
-                            fill = Color.WHITE
-                        }
-                        // Update the icon when the context changes
-                        viewModel.contextProperty.onChange { newContext ->
-                            when(newContext) {
-                                ChapterContext.RECORD -> setIcon(MaterialIcon.MIC_NONE)
-                                ChapterContext.EDIT_TAKES -> setIcon(MaterialIcon.EDIT)
-                                else -> {}
+        pluginOverlay {
+            graphic.apply {
```

Comment:
Minor detail, but could we assign directly to graphic (like you would for a label/button)?